CHANGELOG.md

```diff
@@ -285,6 +316,8 @@
 - improve Channel Creation enforcement
 
 
+[v2.1.1]: https://github.com/BoltzExchange/boltz-client/compare/list...v2.1.1
+[list]: https://github.com/BoltzExchange/boltz-client/compare/v2.1.0...list
```

Comment:
What is list?

Reply:
Fixed. Can you check why the changelog command screwed up here? @jackstar12

Reply:
Had to remove some other stuff too, please test the changelog command again @jackstar12